conference.js

```diff
@@ -1725,15 +1726,20 @@ export default {
             if (user.isHidden()) {
                 return;
             }
+            const displayName = user.getDisplayName();
 
             APP.store.dispatch(participantJoined({
                 id,
-                name: user.getDisplayName(),
+                name: displayName,
                 role: user.getRole()
             }));
 
             logger.log('USER %s connnected', id, user);
-            APP.API.notifyUserJoined(id);
+            APP.API.notifyUserJoined(id, {
+                displayName,
+                formattedDisplayName: appendSuffix(
```

Comment:
What is the point of calling to append a suffix if there is no suffix? Did you originally intent for appendSuffix/formatName to do more?

Reply:
No. So my idea here is to get the exact name that we'll show in the thumbnail.

We can remove the call to appendSuffix. But my intent was to do the same as the DisplayName React component.

Do you want me to remove it? Maybe it safer to leave it if we change the implementation of appendSuffix in future?

Reply:
I really don't care if you remove it or leave it. I wanted to know why you were calling it when it essentially no-ops.